infrastructure/main.tf

```diff
@@ -124,6 +126,7 @@ module "bulk-scan" {
 
     QUEUE_ENVELOPE_SEND = "${data.terraform_remote_state.shared_infra.queue_primary_send_connection_string}"
     QUEUE_NOTIFICATIONS_SEND = "${data.terraform_remote_state.shared_infra.notifications_queue_primary_send_connection_string}"
+    QUEUE_NOTIFICATIONS_READ = "${data.terraform_remote_state.shared_infra.notification_primary_listen_connection_string}"
```

Comment:
should it not be `notifications_queue_primary_listen_connection_string ` instead?
https://github.com/hmcts/bulk-scan-shared-infrastructure/blob/master/queue.tf#L45

Reply:
ah it has changed since this part was coded

Reply:
11 Dec 2018 :)